docs/includes/apiargs-MongoDBCollection-method-watch-option.yaml

```diff
@@ -24,6 +27,16 @@ source:
   ref: maxAwaitTimeMS
 ---
 arg_name: option
+name: readPreference
```

Comment:
We can repeat what is done for `aggregate()`'s `readPreference` option and then use `post`  to add an extra paragraph after the common content:

```
---
source:
  file: apiargs-MongoDBCollection-common-option.yaml
  ref: readPreference
post: |
  This is used for both the initial change stream aggregation and for
  server selection during an automatic resume.
---
```

Sorry if I missed this in an earlier review.